mlflow/models/resources.py

```diff
@@ -137,14 +138,33 @@ def to_dict(self):
     def from_dict(cls, data: Dict[str, str]):
         return cls(function_name=data["name"])
 
+@dataclass
+class DatabricksGenieSpace(DatabricksResource):
+    """
+    Define a Databricks Genie Space to serve a model.
```

Comment:
Can we ad more helpful docstring, such as a link for documentation or example of serving a model in Genie Space? Ideally for other resources as well.

Apparently these docstrings are only documentation we have for those resources, and with current description it's hard to understand what they are for.

Reply:
We currently do not have the documentation yet, I will update it as a follow up when we have all the links

Reply:
Got it, can we add TODO comment and file a tracking jira?